schemas/iso19139.che/src/main/plugin/iso19139.che/formatter/groovy/iso19139che/Handlers.groovy

```diff
@@ -1,6 +1,7 @@
 package iso19139che
 
 import groovy.util.slurpersupport.GPathResult
+import org.fao.geonet.api.records.formatters.groovy.MapConfig
 
 public class Handlers extends iso19139.Handlers {
   public Handlers(handlers, f, env) {
```

Comment:
Sad it is hard to use inheritage power (extends iso19139.Handlers) to avoid code duplication.

Reply:
could not find a better way to do it without modifying the original iso19139 code!

Reply:
Sad we can't modify the original code.